File: 12 Interger to Roman/solution.cpp
class Solution {
public:
    string intToRoman(int num) {
        int value[] = {1000,900,500,400,100,90,50,40,10,9,5,4,1};
        string number[] = {"M","CM","D","CD","C","XC","L","XL","X","IX","V","IV","I"};
        string result = "";
        for (int i=0;i<13;i++){
            while (num>=value[i]){
                num-=value[i];
                result+=number[i];
            }
        }
        return result;
    }
};

File: 12 Interger to Roman/solution.py
class Solution:
    # @param {integer} num
    # @return {string}
    def intToRoman(self, num):
        value = [1000,900,500,400,100,90,50,40,10,9,5,4,1]
        number = ['M','CM','D','CD','C','XC','L','XL','X','IX','V','IV','I']
        result = ''
        for i in range(len(value)):
            while num>=value[i]:
                num-=value[i]
                result+=number[i]
        return result
        